### PR TITLE
Corrected the display of values for the image field.

### PR DESCRIPTION
**Pull request by areksolek**
Corrected the display of values for the image field.

### CODE REVIEW

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #16005 into developer will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/16005)

File: modules/Vtiger/uitypes/MultiImage.php [hunk 1]
```diff
@@ -181,15 +183,15 @@ public function getDisplayValue($value, $record = false, $recordModel = false, $
 			}
 			for ($i = 0; $i < $len; ++$i) {
 				$val = $value[$i];
-				$result .= $val['name'] . ', ';
+				$result .= \App\TextParser::textTruncate($val['name'], $length) . ', ';
```
Comment (rskrzypczak) [inline modules/Vtiger/uitypes/MultiImage.php]: raw - no truncate

File: modules/Vtiger/uitypes/MultiImage.php [hunk 2]
```diff
@@ -227,11 +231,11 @@ public function getListViewDisplayValue($value, $record = false, $recordModel =
 			return '';
 		}
 		$result = '<div class="c-multi-image__result text-center">';
-		foreach ($value as $item) {
+		for ($i = 0; $i < $len; ++$i) {
```
Comment (rskrzypczak) [inline modules/Vtiger/uitypes/MultiImage.php:229]: ?

**Review by rskrzypczak — CHANGES_REQUESTED** [review]

File: modules/Vtiger/uitypes/MultiImage.php [hunk 3]
```diff
@@ -183,20 +185,15 @@ public function getDisplayValue($value, $record = false, $recordModel = false, $
 				$val = $value[$i];
 				$result .= $val['name'] . ', ';
 			}
-			return \App\Purifier::encodeHtml(trim($result, "\n\t ,"));
+			return \App\Purifier::encodeHtml(trim($rawText && $length ? \App\TextParser::textTruncate($result, $length) : $result, "\n\t ,"));
```
Comment (rskrzypczak) [inline modules/Vtiger/uitypes/MultiImage.php]:
```suggestion
			return \App\Purifier::encodeHtml($length ? \App\TextParser::textTruncate($result, $length) : $result, "\n\t ,"));
```

**Pull request merged**